README.md

```diff
@@ -42,13 +43,14 @@ objects like permissions, custom resources and corresponding StatefulSets.
 To create the namespace and apply the operator run the following commands
 
 ```sh
-$ kubectl create namespace dynatrace
-$ kubectl apply -f https://github.com/Dynatrace/dynatrace-operator/releases/latest/download/kubernetes.yaml
+kubectl create namespace dynatrace
```

Comment:
BTW it's very good catch by linter, I saw this issue when try to copy this line via the button in markdown copy code.